### PR TITLE
Bump version to 1.0.1.

### PR DESCRIPTION
**Pull request by markcherbaka**
Learning methodology to push changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bedrock-dev",
-  "version": "1.0.0",
```
Comment (msporny) [inline package.json:3]: Yeah, this needed to be updated, thanks.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "1.0.1",
```
Comment (msporny) [inline package.json:3]: Well, why didn't we upgrade to 1.0.2? I'm not going to merge until I hear back from you.

Comment (markcherbaka) [inline package.json:3]: Because you told me to!  :)

Comment (msporny) [inline package.json:3]: oh yeah, my bad. LGTM.

**Pull request merged**